scripts/client/lint-rust.mjs

```diff
@@ -7,21 +7,24 @@ import {
   workingDirectory,
 } from '../utils.mjs';
 
+const cliArgs = cliArguments();
+const clientPath = cliArgs[0];
+
 // Configure additional arguments here, e.g.:
 // ['--arg1', '--arg2', ...cliArguments()]
 const lintArgs = [
   '-Zunstable-options',
   '--',
   '--deny=warnings',
-  ...cliArguments()
+  ...cliArgs.slice(1)
```

Comment:
I am not too sure on relying on the position of the argument here, since you can use the scripts directly. Ideally we could have this as a flag `--client-path`, but at the same time don't want to complicate the scripts.

Reply:
I wasn't sure which way to go to be honest -- if you prefer `--client-path` or any other approach, I'm game

Reply:
What do you think in following something similar to the programs script, where the script would "detect" all clients from the `Cargo.toml` file?

https://github.com/paladin-bladesmith/sol-stake-view-program/blob/main/scripts/program/format.mjs#L22

Reply:
it's a bit tricky, because `clients` contains more than just rust crates, but I guess we could autodetect based on the presence of a `Cargo.toml` file in the filder

Reply:
I thought we could go over the `members` array from the root `Cargo.toml`.

Reply:
The inverse of this:
```typescript
export function getAllProgramFolders() {
  return getCargo().workspace.members.filter((member) =>
    (getCargo(member).lib?.['crate-type'] ?? []).includes('cdylib')
  );
}
```

Reply:
I went with something like this, as we discussed. Let me know how it looks!

Reply:
Looks great!